keyboards/lets_split/keymaps/default/keymap.c

```diff
@@ -1,4 +1,6 @@
-#include QMK_KEYBOARD_H
+#include "lets_split.h"
```

Comment:
Could you revert the changes to this keymap? 

And ideally, you should create your own folder for the keymap.  Eg: https://docs.qmk.fm/#/newbs_building_firmware

Reply:
ah whoops. I was editing default until i moved everything to keymaps/pyrol